engine/samples/scene/main.cpp

```diff
@@ -28,17 +42,19 @@ using namespace core::data;
 using namespace core::memory;
 using namespace cubos::engine;
 
-void setup(Commands cmds, data::AssetManager& assetManager)
+void setup(World& world, data::AssetManager& assetManager)
 {
+    CUBOS_INFO("SettingUp");
     FileSystem::mount("/assets/", std::make_shared<STDArchive>(SAMPLE_ASSETS_FOLDER, true, true));
 
+    assetManager.registerType<data::Palette>();
     assetManager.registerType<data::Scene>();
     assetManager.importMeta(FileSystem::find("/assets/"));
 
     // Load and spawn the scene.
     auto scene = assetManager.load<data::Scene>("scenes/main");
-    auto root = cmds.create().entity();
-    cmds.spawn(scene->blueprint).add("main", Parent{root});
+    // auto root = world.create().entity();
+    // cmds.spawn(scene->blueprint).add("main", Parent{root});
```

Comment:
Why is this commented?

Reply:
Replacing the Commands with World made those two lines not compile. I just deleted them. If they are needed I'll need some help with changing them to use the World.